docs/whirlpool/docs/03-Whirlpools SDK/03-Position Management/01-Open Position.md

```diff
@@ -0,0 +1,92 @@
+
+## 1. Introduction to Positions in Pools
+
+A position in a liquidity pool represents your contribution of liquidity, which allows traders to swap between tokens while you earn a share of the trading fees. When you open a position, you decide how much liquidity to add, and this liquidity can later be adjusted or removed.
+
+- **Splash Pools**: Provide liquidity without specifying a price range. Ideal for those seeking a simple way to start providing liquidity.
+
+- **Concentrated Liquidity Pools**: Allow you to provide liquidity within a specified price range, enabling higher capital efficiency but requiring more advanced management.
+    > ⚠️ The ratio of Token A to Token B that you deposit as liquidity in the pool is influenced by the current price. As the price moves in either direction, the amounts of Token A and Token B in the pool will adjust. This shift can work to your advantage, but it may also result in the combined value of your tokens (plus any earned rewards) being lower than their initial value before you deposited liquidity. This is commonly referred to as **divergence loss**.
+
+ Upon creation of the position, an NFT will be minted to represent ownership of the position. This NFT is used by the program to verify your ownership when adjusting liquidity, harvesting rewards, or closing the position. 
+ 
+ > ⚠️ Do not burn the position NFT, as burning it will result in the indefinite loss of your position and liquidity.
+
+## 2. Getting Started Guide
+
+Before opening a position, ensure you have completed the environment setup:
+- **RPC Setup**: Use a Solana RPC client to communicate with the blockchain.
+- **Wallet Creation**: Create a wallet to interact with the Solana network.
+- **Devnet Airdrop**: Fund your wallet with a Solana Devnet airdrop to cover transaction fees.
+
+For more details, refer to our [Environment Setup Guide](../01-Environment%20Setup.md)
+
+### Opening a Position in Splash Pools
+
+1. **Pool Address**: Provide the address of the Splash Pool where you want to open a position.
+2. **Liquidity Parameters**: Choose how you want to provide liquidity. You only need to provide one of these parameters, and the function will compute the others in the returned quote based on the current price of the pool:
+    - `liquidity`: Specify the liquidity value to provide.
+    - `tokenA`: Specify the amount of token A (first token in the pool).
+    - `tokenB`: Specify the amount of token B (second token in the pool).
+3. **Slippage Tolerance**: Set the maximum slippage tolerance (optional, defaults to 1%). Slippage refers to the difference between the expected price and the actual price at which the transaction is executed. A lower slippage tolerance reduces the risk of price changes during the transaction but may lead to failed transactions if the market moves too quickly.
```

Comment:
Slippage is currently based on amounts and not on price. i.e. if my expected ratios are:
tokenAEst: 100
tokenBEst: 1000

my max amounts spent will be:
tokenAMax: 101
tokenBMax: 1010

Reply:
Interesting. Is this only the case for positions, or for swaps as well?

Reply:
Yep also swaps